File: exercicios-cpp/estrutura-sequencial/exe11.cpp
#include <iostream>
#include <locale>
#include <iomanip>
#include <cmath> // pow

/*
Faça um Programa que peça 2 números inteiros e um número real. Calcule e mostre:

1) O produto do dobro do primeiro com metade do segundo .
2) A soma do triplo do primeiro com o terceiro.
3) O terceiro elevado ao cubo.
*/

using namespace std;

int main(){
	setlocale(LC_ALL, "portuguese");
	
	int num1=0, num2=0;
	float num3=0.0, resposta_1=0, resposta_2=0, resposta_3=0;
	
	cout << "Informe o primeiro número: ";  cin >> num1;
	cout << "Informe o segundo número: ";  cin >> num2;
	cout << "Informe o terceiro número: ";  cin >> num3;
	
	resposta_1= (num1 *  2) * (num2 / 2);
	resposta_2= (num1 * 3) + num3;
	resposta_3= pow(num3, 3);
	
	cout << fixed << setprecision(2);
	
	cout << "\nO produto do dobro do primeiro com metade do segundo = " << resposta_1;
	cout << "\nA soma do triplo do primeiro com o terceiro = " << resposta_2;
	cout << "\nO terceiro elevado ao cubo = " << resposta_3;
	
	return 0;	
}

File: exercicios-javascript/estrutura-sequencial/exe11.js

/*
Faça um Programa que peça 2 números inteiros e um número real. Calcule e mostre:
o produto do dobro do primeiro com metade do segundo .
a soma do triplo do primeiro com o terceiro.
o terceiro elevado ao cubo.
*/

const readline = require('readline-sync')
const math = require('mathjs')

let num1 = readline.questionInt('\nInforme um numero inteiro: ')
let num2 = readline.questionInt('Informe outro numero inteiro: ')
let num3 = readline.questionFloat('Informe um numero real: ')

let resposta_1 = (num1 * 2) * (num2 / 2)
let resposta_2 = (num1 * 3) + num3
let resposta_3 = math.pow(num3, 3)

console.log(`\nO produto do dobro do primeiro com metade do segundo é ${resposta_1}`)
console.log(`A soma do triplo do primeiro com o terceiro é ${resposta_2}`);
console.log(`O terceiro elevado ao cubo é ${resposta_3}`);
